src/views/pages/Pools/AddLiquidity.js

```diff
@@ -551,21 +551,19 @@ const AddLiquidity = () => {
               </Col>
             </Row>
             <Row className="text-center">
-              <Col xs="12" sm="4">
```

Comment:
Removed the <Col>s wrapping the <Approval>s (see above; handled inside the <Approval> component now)